packages/venia-concept/src/CustomButton/CustomButton.js

```diff
@@ -0,0 +1,73 @@
+import React, { useEffect, useState } from 'react';
+import { oneOf, shape, string, bool } from 'prop-types';
+import { BrowserPersistence } from '@magento/peregrine/lib/util';
+
+import { mergeClasses } from '@magento/venia-ui/lib/classify';
+import defaultClasses from '@magento/venia-ui/lib/components/Button/button.css';
+
+const storage = new BrowserPersistence();
+
+const getRootClassName = (priority, negative) =>
+    `root_${priority}Priority${negative ? 'Negative' : ''}`;
+
+const CustomButton = props => {
+    const {
+        children,
+        classes: propClasses,
+        priority,
+        type,
+        negative,
+        disabled,
+        ...restProps
+    } = props;
```

Comment:
Still need to figure out why these props aren't being passed. I imagine it is something to do with the `replaceJSX` call not passing props. I wasn't sure how to actually forward them with that API.